contract-examples/test/ERC20NativeMinter.ts

```diff
@@ -83,6 +84,13 @@ describe("ERC20NativeMinter", function () {
     await mintTx.wait()
     contractRole = await minterList.readAllowList(contract.address);
     expect(contractRole).to.be.equal(ROLES.MINTER)
+
+    // Enable minter address on TxAllowList to allow deposits
+    const allowList = await ethers.getContractAt("IAllowList", TX_ALLOW_LIST_ADDRESS, owner)
+    let mintEnableTx = await allowList.setEnabled(minter.address);
+    await mintEnableTx.wait()
+    contractRole = await allowList.readAllowList(minter.address);
+    expect(contractRole).to.be.equal(ROLES.MINTER)
```

Comment:
these tests should not contain any other precompiles. Tests are not in perfect shape currently (i.e they're not stateless) but at least we should keep them separate from other precompiles.